app/DynamicsAdapter/DynamicsAdapter.sln

```diff
@@ -11,9 +11,14 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test", "test", "{1BE19189-0
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DynamicsAdapter.Web.Test", "DynamicsAdapter.Web.Test\DynamicsAdapter.Web.Test.csproj", "{6283EFDD-21EC-4FE5-B01E-54588BBB8FCD}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Fams3Adapter.Dynamics", "Fams3Adapter.Dynamics\Fams3Adapter.Dynamics.csproj", "{64CD5D00-7677-4F3B-BBAA-8F70B68DB4A7}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Fams3Adapter.Dynamics", "Fams3Adapter.Dynamics\Fams3Adapter.Dynamics.csproj", "{64CD5D00-7677-4F3B-BBAA-8F70B68DB4A7}"
```

Comment:
what's that?

Reply:
> looks good. Just not sure if sln, proj file can be checked in.

When project file have some changes they will be checked in. For example when we increase version number on contract. 

Is there a build issue on your end when you pull the branch and run it?

Reply:
checked. it can build.